variables.tf

```diff
@@ -246,6 +246,12 @@ variable "scaling_configuration" {
   default     = {}
 }
 
+variable "serverlessv2_scaling_configuration" {
+  description = "Map of nested attributes with serverless v2 scaling properties. Only valid when `engine_mode` is set to `provisioned`"
+  type        = map(string)
```

Comment:
Shouldn't we add the proper config options instead just a map? This kills autocompletion in IDEs.

Reply:
There are so many limitations of `object` type (e.g. optional keys) that it does not make sense to use it in these modules.

Reply:
Ok, makes sense